### DISCUSSION TITLE
Espers [ESP] Support

### DISCUSSION THREAD

**Discussion by jkdi** [coinfoundry/miningcore#1399] [Ideas]
Hello,
is it possible to add Espers Blockchain support to mine ESP? 

https://github.com/CryptoCoderz/Espers

its a hmq1725 algo.

thx!

**Comment by oliverw** [comment]
Everything is possible. It's just someone has to do it.

**Reply by jkdi** [reply]
sorry, i'm not a programmer, but i can ask in the espers telegram group!? is this ok for you?

**Reply by oliverw** [reply]
Of course.
